source/event/2017/toronto-showcase.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "IIIF Showcase - Toronto - 2017"
+layout: spec
+tags: [event ]
+---
+
+## Table of Contents
+{:.no_toc}
+
+* Goes Here
+{:toc}
+
+## IIIF Showcase - Toronto - 2017
+
+Join the International Image Interoperability Framework (IIIF) community for an evening showcase to demonstrate the power and potential of interoperable image delivery on the Web. Embraced by large and growing number of the world’s leading research and cultural heritage organizations, IIIF provides an open framework for publishing digital image-based resources, to be viewed, shared, cited, annotated, and more by any compatible image-viewing application. The program will provide an introduction to IIIF, exploring how institutions are leveraging IIIF to improve image delivery solutions, while simultaneously providing new and rich features and functionality for a world-class end user experience. As a community-driven initiative, we welcome anyone with an interest to participate in this growing movement and take advantage of the common framework.
```

Comment:
Embraced by *a* large and growing number -- missing "*a*"

Reply:
thank you!